apps/docs/src/app/platform/component-docs/platform-dynamic-page/platform-dynamic-page-examples/platform-dynamic-page-snap-scroll-example.component.scss

```diff
@@ -0,0 +1,4 @@
+.scrollable-container {
+    height: 20rem;
+    overflow-y: scroll;
+}
```

Comment:
new line required

Reply:
`ng lint platform` passes for this and expected newline to be removed, so it was removed.